config/install/migrate_plus.migration.media_existing.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - localgov_media
+id: media_existing
+migration_group: localgov_migration
+label: 'Media - existing'
+source:
+  plugin: d8_entity
+  entity_type: media
+process:
+  mid: mid
+  vid: vid
+  status: status
+  uid: uid
+  uuid: uuid
+  name: name
+  created: created
+  changed: changed
+  bundle: bundle
```

Comment:
This migration is going to cause problems for BHCC. It assumes that the Media bundle already exists in the LocalGov site. This is fine for Croydon where the two media bundles in use match those in LocalGov (`document` and `media`), but the DB I have from BHCC has a `news_default_image` bundle.

Ideally, there should be a check that the bundle is one of those included with LocalGov, but perhaps @andybroomfield can say whether this is likely to be a problem for him.

Reply:
Good point.  I will add a check.  This is why you need code reviews :)

Reply:
I have just added a restriction for the imported bundle types.  Only the *image*, *document*, and *remote_video* bundles are now processed as these are [declared](https://github.com/localgovdrupal/localgov_core/tree/master/modules/localgov_media/config/optional) in the localgov_core:localgov_media submodule.